src/parcel.F

```diff
@@ -2447,55 +2448,55 @@ subroutine write_prclctl(name_prcl,desc_prcl,unit_prcl,prec)
       end subroutine write_prclctl
 
       real function calcW1(rx,ry,rz)
-      !$acc routine vector 
+      !$acc routine vector
```

Comment:
I wanted to leave those 'routine vector' in the code because I did not want to cause conflicts with David's merge.  I can easy clean these up now that David's changed have gone in.

Reply:
That makes sense. Hope we remember to revisit this code after David's PR.